CareKitUI/CareKitUI/iOS/Task/Collection/OCKSliderStyle.swift

```diff
@@ -0,0 +1,46 @@
+//
+//  OCKSlider.swift
+//
+//
+//  Created by Dylan Li on 8/03/20.
+//  Copyright © 2020 NetReconLab. All rights reserved.
+//
+
+#if !os(watchOS)
+
+import SwiftUI
+
+/// The style of an OCK Slider.
+public enum OCKSliderStyle {
+    
+    /// An OCKSlider style with a bar shaped body and tick marks. The dimensions of the slider  are determined by an instance of OCKSliderDimensions.
+    ///
+    /// ```
+    ///      |      ,      ,      ,      ,      |
+    ///      +––––––––––––––––––––––––––––––––––+
+    ///     |                                    |
+    ///     |                                    |
+    ///      +––––––––––––––––––––––––––––––––––+
+    ///      |      '      '      '      '      |
+    /// ```
+    case CareKitSlider(OCKSliderDimensions)
+    
+    /// An OCK Slider that uses the body of a system slider.
+    case UISlider
+}
+
+/// The dimensions that determine the size and frame of an OCKSlider with a filler style
+public struct OCKSliderDimensions {
```

Comment:
We can use reasonable defaults instead of making this a configurable part of the API.